vendor/bat-native-ledger/src/bat/ledger/internal/promotion/promotion_util.cc

```diff
@@ -246,6 +255,101 @@ std::unique_ptr<base::ListValue> ParseStringToBaseList(
   return std::make_unique<base::ListValue>(value->GetList());
 }
 
+bool UnBlindTokens(
+    ledger::PromotionPtr promotion,
+    std::vector<std::string>* unblinded_encoded_tokens,
+    std::string* error) {
+  if (
+      !promotion ||
+      !promotion->credentials ||
+      !unblinded_encoded_tokens ||
+      !error) {
```

Comment:
Seems like a logic error here. If `!error` is `true`, we don't want to dereference it.

Reply:
fixed